src/main/scala/cromwell/binding/command/Command.scala

```diff
@@ -54,7 +54,7 @@ object Command {
 case class Command(parts: Seq[CommandPart]) {
   val ws = Pattern.compile("[\\ \\t]+")
   def inputs: Seq[TaskInput] = {
-    parts.collect {case p: ParameterCommandPart => p}.map {p =>
+    val collectedInputs = parts.collect {case p: ParameterCommandPart => p}.map {p =>
```

Comment:
Nope you're good, scalac apparently is smarter with the `collect` than I thought it was.  :smile: